cms/templates/partials/metadata-tiles.html

```diff
@@ -40,10 +40,11 @@
         <span class="title">FORMAT</span>
         <span class="text">Online</span>
       </li>
-      {% if page.product.current_price %}
+      {% firstof page.product.first_unexpired_run.current_price page.product.current_price as price%}
```

Comment:
@mbertrand Could you please take a look at this PR. Is this a right way to get appropriate price for a course ?

Reply:
This seems to be the best way possible given the current functions in the code.  However it is possible that the price shown may be for a course run that doesn't match the next start date.  For example I created 3 course runs for a course, with start dates in 2010, 2013, and 2020, no end dates (so none had expired) and product prices of $25, $50, and $150.   The course page shows the next start date as 2020 but shows the price for the 2010 run ($25).  This is because the code is using `next_start_date()` to get the earliest date of a run that has not yet started, but using `first_unexpired_run()` to get the price.  To be consistent, would need to create and use a `next_run()` function to get the price, or create and use a `first_unexpired_run_date()` function for the date.  

Not sure how likely it is that the above scenario might ever exist in production, but my guess is it might be best to make sure the price and date shown are consistent with each other just in case?

Reply:
<img width="714" alt="Screen Shot 2019-05-31 at 8 56 36 AM" src="https://user-images.githubusercontent.com/187676/58707970-6c746580-8384-11e9-8d9e-fdc193bacfd3.png">

Reply:
@mbertrand thanks for such detailed feedback.

Reply:
@mbertrand Does this make sense to use `page.product.first_unexpired_run.date` and `page.product.first_unexpired_run.current_price` for `date` and `price` respectively ?

Reply:
That should work for a course page, for a program page you'd probably have to add this to the `Program` model:

```python
    @property
    def first_unexpired_run(self):
        """Gets the earliest start date of unexpired CourseRuns if one exists"""
        return min(
            filter(None, [course.first_unexpired_run for course in self.courses.all()]),
            default=None,
        )
```

@Ferdi or @pdpinch might have a preference on which to show: the next future run or the first unexpired run.  But I'm okay with choosing first_unexpired_run for now and it can be changed later if necessary.

Reply:
@mbertrand Feedback addressed.

Reply:
@mbertrand I have a question about program `start_date`. In a scenario where there is a program with couple of courses and respective runs. If first `course_run` is expired, program will still show `start_date` of the next unexpired course run, which could be wrong and not as we expect.
What would you suggest ? Is it ok to show `start_date` of `first_unexpired_run` ?
CC: @pdpinch

Reply:
Ideally, on the program page, we would display the start date of the next _run_ of the program, which is the start date of the courserun of the _first_ course in the program (not any course). 

However,  I think this is out of scope for this PR.

Reply:
@pdpinch thanks, I will merge this after getting approval from @mbertrand. 

If we need to change the displayed date of the program we can handle it in a separate issue.